src/Uno.Extensions.Templates/content/unoapp-extensions/Directory.Packages.props

```diff
@@ -55,25 +61,36 @@
     <PackageVersion Include="Uno.Extensions.Serialization" Version="255.255.255.255" />
     <PackageVersion Include="Uno.Extensions.Serialization.Http" Version="255.255.255.255" />
     <PackageVersion Include="Uno.Extensions.Serialization.Refit" Version="255.255.255.255" />
-    <PackageVersion Include="Uno.Material.WinUI" Version="2.3.0" />
+    <!--#if (use-material)-->
+    <PackageVersion Include="Uno.Material.WinUI" Version="2.4.0-dev.68" />
+    <!--#elif (use-cupertino)-->
+    <PackageVersion Include="Uno.Cupertino.WinUI" Version="2.4.0-dev.68" />
+    <!--#endif-->
     <PackageVersion Include="Uno.Toolkit.WinUI" Version="2.4.0-dev.129" />
+    <!--#if (use-material)-->
     <PackageVersion Include="Uno.Toolkit.WinUI.Material" Version=" 2.4.0-dev.129" />
-    <PackageVersion Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.6.19" />
+    <!--#endif-->
+    <PackageVersion Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.7.0-dev.564" />
     <PackageVersion Include="Uno.UITest.Helpers" Version="1.1.0-dev.32" />
     <PackageVersion Include="Uno.UniversalImageLoader" Version="1.9.36" />
-    <PackageVersion Include="Uno.Wasm.Bootstrap" Version="7.0.8" />
-    <PackageVersion Include="Uno.Wasm.Bootstrap.DevServer" Version="7.0.8" />
-    <PackageVersion Include="Uno.WinUI" Version="4.6.19" />
-    <!--#if (markup)-->
+    <PackageVersion Include="Uno.Wasm.Bootstrap" Version="7.0.11" />
+    <PackageVersion Include="Uno.Wasm.Bootstrap.DevServer" Version="7.0.11" />
+    <PackageVersion Include="Uno.Wasm.Bootstrap.Server" Version="7.0.11" />
+    <PackageVersion Include="Uno.WinUI" Version="4.7.0-dev.564" />
+    <!--#if (use-csharp-markup)-->
     <PackageVersion Include="Uno.WinUI.Markup" Version="4.6.0-dev.13" />
     <PackageVersion Include="Uno.Toolkit.WinUI.Markup" Version="2.4.0-dev.129" />
+    <!--#if (use-material)-->
+    <PackageVersion Include="Uno.Material.WinUI.Markup" Version="2.4.0-dev.68" />
+    <!--#endif-->
+    <PackageVersion Include="Uno.Themes.WinUI.Markup" Version="2.4.0-dev.68" />
     <!--#endif-->
-    <PackageVersion Include="Uno.WinUI.RemoteControl" Version="4.6.19" />
-    <PackageVersion Include="Uno.WinUI.Skia.Gtk" Version="4.6.19" />
-    <PackageVersion Include="Uno.WinUI.Skia.Linux.FrameBuffer" Version="4.6.19" />
-    <PackageVersion Include="Uno.WinUI.Skia.Wpf" Version="4.6.19" />
-    <PackageVersion Include="Uno.WinUI.WebAssembly" Version="4.6.19" />
-    <PackageVersion Include="Xamarin.Google.Android.Material" Version="1.4.0.4" />
-    <PackageVersion Include="Xamarin.UITest" Version="3.2.6" />
+    <PackageVersion Include="Uno.WinUI.RemoteControl" Version="4.7.0-dev.564" />
```

Comment:
Are we still pushing to single location? If not, can we revert Uno refs to latest stable

Reply:
no we will not be targeting latest stable as there should still be some updated targets in the 4.7 dev build... tbd once the updated template format is finalized